linters/index.js

```diff
@@ -0,0 +1,10 @@
+// Copyright 2018 VMware, Inc.
```

Comment:
It's little vague for me why do we need this file: linters/index.js?
Also if this is a config file shouldn't the name be something like: config_linters or just config?

Reply:
So when you include the linters folder in the runner file you basically include linters/index.js?

Reply:
This decouples the runner code from specific linters: all that is necessary to enable or disable specific linters is to modify the fields in this object.

Reply:
> So when you include the linters folder in the runner file you basically include linters/index.js?

Yes, importing a folder imports the `index.js` file if it exists.

Reply:
But why the name of the file is index.js?
I think it would be more clearer if it is something like the above suggestions I gave.

Maybe this naming is some kind of JavaScript pattern but I don't know.

Reply:
So we can just write `requires('./linters')` instead of `requires('./linters/default_linters')` (makes more sense to me)

Reply:
Ookay, maybe you are right. It's not such a big deal. :D